### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by ghodsehimanshu17295665**
1] Created a Models for - TimeStampedModel, User, Task, Comment
2] Register Model in admin file
3] Created a forms - SignupForms, LoginForms, TaskForm, CommentForm, TaskStatusForm
4] Created Views
5] Created urls
6] Create Templates files

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils import timezone
+
+
+class TimeStampedModel(models.Model):
```
Comment (sagarjain2104) [inline app/models.py]: Django provides the TimeStamped model. Find and replace it

File: app/views.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        form = TaskForm(request.POST)
+        if form.is_valid():
+            task = form.save(commit=False)
+            task.creator = request.user
+            task.save()
+
+            # Send email
+            assignee = task.assignee
+            subject = f"New Task Assigned: {task.title}"
+            message = f"You have been assigned a new task: {task.title}\nDescription: {task.description}\nPriority: {task.get_priority_display()}\nDue_Date: {task.due_date}"
```
Comment (sagarjain2104) [inline app/views.py]: create a function for sending emails in utils.py and use that function here

File: app/models.py (hunk 1 above)
Comment (ghodsehimanshu17295665) [reply app/models.py]: Okay Sir..

File: app/views.py (hunk 2 above)
Comment (ghodsehimanshu17295665) [reply app/views.py]: Okay Sir.

**Review by sagarjain2104 — APPROVED** [review]
Looks Good

**Pull request merged**